### PR TITLE
Ensure training is in line with production for postgres version

### PR DESCRIPTION
**Pull request by acsauk-mojd**
## Purpose
Because workspace != account

### CODE REVIEW

**Review by jamesrwarren — APPROVED** [review]

**Review by coderste — DISMISSED** [review]

File: environment/api_rds.tf [hunk 1]
```diff
@@ -18,8 +18,8 @@ resource "aws_db_instance" "api" {
   license_model               = "postgresql-license"
   maintenance_window          = "sun:01:00-sun:02:30"
   monitoring_interval         = "0"
-  option_group_name           = terraform.workspace == "production02" ? "default:postgres-9-6" : "default:postgres-10"
-  parameter_group_name        = terraform.workspace == "production02" ? "default.postgres9.6" : "default.postgres10"
+  option_group_name           = contains(["production02", "training"], terraform.workspace) ? "default:postgres-9-6" : "default:postgres-10"
+  parameter_group_name        = contains(["production02", "training"], terraform.workspace) ? "default.postgres9.6" : "default.postgres10"
```
Comment (TomTucka) [inline environment/api_rds.tf:22]: If we remove these RDS will just pick the default

Comment (acsauk-mojd) [reply environment/api_rds.tf:22]: Do you mean `option_group_name` and `parameter_group_name` @TomTucka ?

Comment (TomTucka) [reply environment/api_rds.tf:22]: Yep 🙂

Comment (acsauk-mojd) [reply environment/api_rds.tf]: Nice - dropped now 👍

**Review by jamesrwarren — APPROVED** [review]

**Pull request merged**